docs/docs/multipart.md

```diff
@@ -0,0 +1,261 @@
+# Multipart and Form Handling
+
+Multipart is a content type for messages (requests or responses) composed of multiple parts.
+Each part is itself similar to a message in that it has its own body and headers. A common use
+for multipart requests is for user-submitted forms, especially ones that include files.
+Browsers also support (and default to) another encoding for forms, `application/x-www-form-urlencoded`.
+This encoding is simpler, but is not often used for binary data.
+
+## UrlForm
+
+To handle `application/x-www-form-urlencoded` messages, http4s provides `UrlForm` and respective `EntityEncoder` and 
+`EntityDecoder`. The following example shows the client sending a form request and a server parsing it:
+
+```scala mdoc:silent
+import org.http4s.client.Client
+import cats.effect._
+import org.http4s._
+import org.http4s.dsl.io._
+import org.http4s.headers._
+import org.http4s.multipart._
+import org.http4s.implicits._
+
+import cats.effect.unsafe.implicits.global
```

Comment:
Because code snippets are often copy pasted into projects, and this is only suitable in documentation or a repl, I'd like to see this get special handling as we do in [the client.md documentation](https://github.com/http4s/http4s/blame/b7978517dd6fb871f26caf30c00c2766d3c8bea4/docs/docs/client.md#L88-L110).

Reply:
You mean just the `implicit val runtime: IORuntime = IORuntime.global` instead of the import, right?
The allocation bit wouldn't be necessary because i'm never running a full server (except in the scala-cli bit, which is self-contained)

Reply:
No, I mean both the import and the creation of the value.
Users often copy imports, they shouldn't be copying the `unsafe.implicits.global` import accidentally.

That's why I recommend the approach in [the client.md documentation](https://github.com/http4s/http4s/blame/b7978517dd6fb871f26caf30c00c2766d3c8bea4/docs/docs/client.md#L88-L110).
It isolates the import with clear guidance.
To be clear, mdoc really is executing the code in the `@:callout(info)` section, so by putting it there we can remove it from this top level list of imports.

Either

```
import cats.effect.unsafe.IORuntime
implicit val runtime: IORuntime = IORuntime.global
```

or just

```
import cats.effect.unsafe.implicits.global
```

works fine

Reply:
> To be clear, mdoc really is executing the code in the @:callout(info) section, so by putting it there we can remove it from this top level list of imports.

I didn't know this! I'll fix it